utilities/constants.py

```diff
@@ -122,6 +127,27 @@ class StorageClassName:
     NFS: str = "nfs"
 
 
+class DscComponents:
+    MODELMESHSERVING: str = "modelmeshserving"
+    KSERVE: str = "kserve"
+    MODELREGISTRY: str = "modelregistry"
+
+    class ManagementState:
+        MANAGED: str = "Managed"
+        REMOVED: str = "Removed"
+
+    class ConditionType:
+        MODEL_REGISTRY_READY: str = "ModelRegistryReady"
+        KSERVE_READY: str = "KserveReady"
+        MODEL_MESH_SERVING_READY: str = "ModelMeshServingReady"
```

Comment:
Aren't these supposed to be `kserveReady`, `model-meshReady` and `model-registry-operatorReady`?

Reply:
not in 2.17